Base/usr/share/man/man1/notify.md

```diff
@@ -0,0 +1,19 @@
+## Name
+
+notify - create a nofitication
+
+## Synopsis
+
+```**sh
+$ notify <title> <message> [icon-path]
```

Comment:
Current convention is to add an `Arguments` heading for positional arguments and an `Options` heading for command line options.

```
$ grep -rn '## Arguments' Base/usr/share/man/
Base/usr/share/man/man1/jp.md:22:## Arguments
Base/usr/share/man/man1/TextEditor.md:20:## Arguments:
Base/usr/share/man/man1/Inspector.md:11:## Arguments
Base/usr/share/man/man1/ls.md:35:## Arguments
Base/usr/share/man/man1/Profiler.md:30:## Arguments
Base/usr/share/man/man1/Playground.md:11:## Arguments
Base/usr/share/man/man1/QuickShow.md:15:## Arguments
Base/usr/share/man/man8/sysctl.md:21:## Arguments
```

```
$ grep -rn '## Options' Base/usr/share/man/
Base/usr/share/man/man1/echo.md:16:## Options
Base/usr/share/man/man1/gml-format.md:15:## Options
Base/usr/share/man/man1/test.md:81:## Options
Base/usr/share/man/man1/date.md:16:## Options
Base/usr/share/man/man1/Shell.md:25:## Options
Base/usr/share/man/man1/md.md:18:## Options
Base/usr/share/man/man1/base64.md:16:## Options
Base/usr/share/man/man1/jp.md:17:## Options
Base/usr/share/man/man1/watch.md:16:## Options
Base/usr/share/man/man1/Terminal.md:15:## Options
Base/usr/share/man/man1/readlink.md:15:## Options
Base/usr/share/man/man1/adjtime.md:24:## Options
Base/usr/share/man/man1/js.md:22:## Options
Base/usr/share/man/man1/uname.md:16:## Options
Base/usr/share/man/man1/xargs.md:30:## Options
Base/usr/share/man/man1/syscall.md:15:## Options
Base/usr/share/man/man1/tar.md:18:## Options
Base/usr/share/man/man1/test-js.md:24:## Options
Base/usr/share/man/man1/TextEditor.md:16:## Options
Base/usr/share/man/man1/expr.md:60:## Options
Base/usr/share/man/man1/find.md:21:## Options
Base/usr/share/man/man1/mktemp.md:18:## Options
Base/usr/share/man/man1/mkdir.md:15:## Options
Base/usr/share/man/man1/crash.md:17:## Options
Base/usr/share/man/man1/ls.md:17:## Options
Base/usr/share/man/man1/pmap.md:15:## Options
Base/usr/share/man/man1/Profiler.md:26:## Options
Base/usr/share/man/man1/pape.md:16:## Options
Base/usr/share/man/man1/avol.md:15:## Options
Base/usr/share/man/man1/EchoServer.md:15:## Options
Base/usr/share/man/man8/userdel.md:17:## Options
Base/usr/share/man/man8/useradd.md:19:## Options
Base/usr/share/man/man8/TelnetServer.md:16:## Options
Base/usr/share/man/man8/sysctl.md:17:## Options
Base/usr/share/man/man8/purge.md:16:## Options
Base/usr/share/man/man5/SystemServer.md:19:## Options
```

Reply:
Fixed.